### PR TITLE
...

### PR DESCRIPTION
**Pull request by nazo44**

### CODE REVIEW

**Comment by blackrainbowtest** [comment]
Functionality - Everything meets the requirements of the task.
Complexity - The project does not contain complex elements.
Naming - Naming corresponds to meaning.
Good Things - Random function used.

Recommendations:
1) Instead of empty prints, you can use "\n" for line breaks.
2) Add error handlers to avoid program crash (checking the correct type of input data). This can be either a simple data type checking method (for this project) or a more complex design for future projects.

**Comment by derenikvardanyan** [comment]
The overall design — There is no any issue with this criteria.
Functionality — Functionally the code works fine.
Complexity — Tried to write the code as optimal as possible, good!
Naming — The name matches the content.
Good Things — 1) randint() method from random module is used, 2) Break and continue construction is used.

Recommendations` Python has also "Triple Quotes" way of writinh strings when you have a multiline text. Would be helpful to know for the future future project.
https://www.scaler.com/topics/python-triple-quote/

In conclusion: Good Job, thank you!
